app/assets/javascripts/current_deploys.js

```diff
@@ -0,0 +1,31 @@
+$(function() {
+  var $badge = $('.current-deploys .badge');
+  $error     = $("#error_message");
```

Comment:
I'd really appreciate it if you could align this properly :) i.e.
```js
var $badge = ...,
    $error = ...;
```
note the comma and the semi colon. (unless the `$error` is from somewhere else and you meant to do that)